# sensor params
sensor:
  image_dir: data/own
  type: virtual_primesense
  frame: primesense_overhead
calib_dir: data/calib

# policy params
policy:
  # optimization params
  num_seed_samples: 256
  num_gmm_samples: 64
  num_iters: 3
  gmm_refit_p: 0.25
  gmm_component_frac: 0.4
  gmm_reg_covar: 0.01

  # general params
  deterministic: 1
  gripper_width: 0.05
  
  # sampling params
  sampling:
    # type
    type: suction

    # params
    max_suction_dir_optical_axis_angle: 45
    delta_theta: 1
    delta_phi: 1
    mean_depth: 0.001
    sigma_depth: 0.001
    min_suction_dist: 1.0
    angle_dist_weight: 5.0
    depth_gaussian_sigma: 1.0

    max_dist_from_center: 250
    max_num_samples: 10000

    num_grasp_samples: 500

  # metrics
  metric:
    type: gqcnn
    gqcnn_model: /nfs/diskstation/projects/dex-net/bin_picking/models/suction/gqcnn_finetuned_01_08_18

    crop_height: 96
    crop_width: 96

  # visualization
  vis:
    grasp_sampling : 0
    tf_images: 0
    grasp_candidates: 1
    elite_grasps: 0
    grasp_ranking: 0
    grasp_plan: 0
    final_grasp: 1

    k: 25

# image proc params
inpaint_rescale_factor: 0.5
